src/components/DataDisplay/Tutorial/Tutorial.tsx

```diff
@@ -0,0 +1,69 @@
+import { forwardRef, ReactElement, useState } from 'react';
+
+import { useTutorialSteps } from '../../../providers/TutorialStepsProvider';
+import TutorialStart from './TutorialStart';
+import TutorialSteps from './TutorialSteps';
+
+export enum Steps {
+  NOTSTARTED = 'not-started',
+  STEPONE = 'step-one',
+  STEPTWO = 'step-two',
+  LASTSTEP = 'last-step',
+}
+
+export interface IStep {
+  name: Steps;
+  title: string;
+  body: ReactElement;
+  button: string;
+}
+
+export type TutorialProps = {
+  steps: IStep[];
+  starterTitle: string;
+  imageSource?: string;
```

Comment:
I think there should be a boolean value here for if it's open or not, so we avoid having the first step being 'not-started' instead of it actually being a step, if that makes sense 😅